src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -107,12 +111,13 @@ public function startApp()
         if (!isset($config['class'])) {
             $config['class'] = 'yii\web\Application';
         }
-
+        if (!isset($config['container']['definitions']['yii\log\Logger']['class'])) {
+            $config['container']['definitions']['yii\log\Logger']['class'] = 'Codeception\Lib\Connector\Yii2\Logger';
```

Comment:
what if `$config['container']['definitions']['yii\log\Logger']` is not set?
this goes to break the stuff. Also, I don't like how hardcoded it is here...